packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -147,6 +149,84 @@ class InteractiveViewer extends StatefulWidget {
        constrained = false,
        child = null;
 
+  /// When laying out the child InteractiveViewer applies a transform based on the current
+  /// [Matrix4] defined in the [TransformationController] and will translate all the hit tests as well.
+  ///
+  /// Due to the way gestures work in Flutter if a child is in an negative offset of it's parent then
+  /// hit tests are skipped. For example you could have a Stack with clip none and negative offset
+  /// for the [Positioned] widget.
+  ///
+  /// ```dart
+  /// InteractiveViewer(
```

Comment:
Apparently the `Linux analyze` task fails on issues with the code sample.
(see https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8734698621183606385/+/u/run_test.dart_for_analyze_shard_and_subshard_None/stdout)

And the DartDoc generator also encountered an issue on your change.
(see https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8734698621171343873/+/u/run_test.dart_for_docs_shard_and_subshard_None/stdout)

Reply:
Pushed out some commits that should fix it. I thought true/false would be picked up as a type.